include/LocklessList.h

```diff
@@ -95,7 +85,7 @@ class LocklessList
 
 
 private:
-	QAtomicPointer<Element> m_first;
+    std::atomic<Element*> m_first;
```

Comment:
Should be tab-indented (not spaces) for consistency.

Reply:
Doi. So much for having my vim settings fixed up.